src/uiprotect/api.py

```diff
@@ -829,15 +829,21 @@ async def update(self) -> Bootstrap:
         async with self._update_lock:
             bootstrap = await self.get_bootstrap()
             if bootstrap.nvr.version >= NFC_FINGERPRINT_SUPPORT_VERSION:
-                bootstrap.keyrings = cast(
-                    dict[str, Keyring],
-                    dict_from_unifi_list(self, await self.api_request_list("keyrings")),
+                bootstrap.keyrings.replace_with_list(
+                    cast(
+                        list[Keyring],
+                        list_from_unifi_list(
+                            self, await self.api_request_list("keyrings")
+                        ),
+                    )
                 )
-                bootstrap.ulp_users = cast(
-                    dict[str, UlpUser],
-                    dict_from_unifi_list(
-                        self, await self.api_request_list("ulp-users")
-                    ),
+                bootstrap.ulp_users.replace_with_list(
```

Comment:
We are going to need to make a new object each time or else it will mutate the classvar and multiple instances will end up with the same object

Reply:
@bdraco done